decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -241,7 +240,7 @@ def default_filter_scope_params
         if current_component.participatory_space.scope
           ["all", current_component.participatory_space.scope.id] + current_component.participatory_space.scope.children.map { |scope| scope.id.to_s }
         else
-          %w(all global) + current_component.participatory_space.scopes.map { |scope| scope.id.to_s }
+          %w(all global) + current_component.participatory_space.scopes.pluck(:id)
```

Comment:
To keep the original behavior (just in case):

```suggestion
        ["all"] + current_component.participatory_space.categories.pluck(:id).map(&:to_s)
```